src/node_api.cc

```diff
@@ -123,6 +187,22 @@ v8::Local<v8::Value> V8LocalValueFromJsValue(napi_value v) {
   return local;
 }
 
+napi_status V8NameFromPropertyDescriptor(napi_env env,
```

Comment:
I understand the benefit of `static inline` but why do you prefer an anonymous namespace? I think there's some value in clumping together the v8 helpers and naming that clump `v8impl` (though name could be more descriptive 👍). We might have other namespaces for other clumps of helpers like `uvimpl`, etc so having a relevant name helps to reason about these groups, no?